app/errors.py

```diff
@@ -0,0 +1,14 @@
+class VaccineError(Exception):
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    pass
+
+
+class OutdatedVaccineError(VaccineError):
+    pass

```

Comment:
Consider adding a constructor to the `OutdatedVaccineError` class to accept a custom error message. This will make the error more descriptive and helpful for debugging.